e2e/rbd.go

```diff
@@ -115,6 +115,16 @@ var _ = Describe("RBD", func() {
 			By("create a PVC and Bind it to an app with normal user", func() {
 				validateNormalUserPVCAccess(pvcPath, f)
 			})
+			// Skipping ext4 FS testing
```

Comment:
Nit: it doesn't look like the test is being skipped anymore, so this comment is out-of-date